tests/robot-tests/tests/admin/bau/create_methodology_for_publication.robot

```diff
@@ -18,4 +18,14 @@ teardown suite
 Create Methodology for Publication
     [Tags]  HappyPath
     user creates test publication via api   ${PUBLICATION_NAME}
-    user creates methodology for publication    ${PUBLICATION_NAME}
\ No newline at end of file
+    ${accordion}=  user opens publication on the admin dashboard   ${PUBLICATION_NAME}
+    user checks element contains button  ${accordion}  Create methodology
+    user checks element contains button  ${accordion}  Link to an externally hosted methodology
+    user creates methodology for publication    ${PUBLICATION_NAME}
+    ${accordion}=  user opens publication on the admin dashboard   ${PUBLICATION_NAME}
+    user checks element does not contain button  ${accordion}  Create methodology
+    user checks element does not contain button  ${accordion}  Link to an externally hosted methodology
+    user views methodology for publication    ${PUBLICATION_NAME}
```

Comment:
`user views methodology for publication` is calling `user opens publication on the admin dashboard` unnecessarily. I suspect you already knew this when you wrote this, and I too don't think its that big of a deal so maybe leave this. Just pointing it out in case you weren't aware.

Reply:
I think I will change it as you're right that it doesn't need to be done here.